0010-self-service/0010-self-service.md

```diff
@@ -0,0 +1,7 @@
+# Self service
+
+As a potential user I would like to create k8s clusters and/or my own control plane without having to talk to Giant Swarm.
```

Comment:
I was a bit confused first, cause we have also talked about widening our user-base within customers to have more people and especailly also end-user developers have access to GS as a product and interface, but this is for non-GS customers, right? So more of a trial or entry approach.

Reply:
On a high level not necessarily. Could also be the engineering teams not having to talk to the platform team that has access to happa.

Reply:
Sure, I understand where you are coming from on a high abstraction level. I would most probably however separate those two opportunities to give room to more ideas and solutions towards them. E.g. the solutions one could apply to the "engineering teams not having to talk to the platform team that has access to GS" opportunity do not necessarily also apply to or solve all the requirements of the wider non-GS opportunity (vice verse might apply though).